.vm_provision/playbook/roles/xdoc/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Clone xdoc source code
```

Comment:
This is the only bit that really bugs me: right now to get this going, you have to clone the repo onto your host, only for it to be then cloned *again* in the VM. That's inefficient, both of time and of storage.

Worse: it doesn't work twice. If you try to reprovision the VM, this will fail. 

A few ideas here...

- Provide a script to pull down JUST the vagrantfile + provisioning scripts from the repo (faster to get going)
- Instead of cloning here, just copy everything from `/vagrant` (but ONLY IF `~/code` doesn't already exist) then git fetch / pull to update it.
- Just work entirely in `/vagrant`? More notes on that later...

Reply:
@josh-heyer thanks a lot for the feedback, let me address them and submit changes.

Reply:
I've removed this, see the newest commit